lib/service/Service.ts

```diff
@@ -58,6 +59,8 @@ const argChecks = {
   POSITIVE_AMOUNT: ({ amount }: { amount: number }) => { if (amount <= 0) throw errors.INVALID_ARGUMENT('amount must be greater than 0'); },
   POSITIVE_QUANTITY: ({ quantity }: { quantity: number }) => { if (quantity <= 0) throw errors.INVALID_ARGUMENT('quantity must be greater than 0'); },
   PRICE_NON_NEGATIVE: ({ price }: { price: number }) => { if (price < 0) throw errors.INVALID_ARGUMENT('price cannot be negative'); },
+  QUANTITY_CORRECT_PRECISION: ({ quantity }: { quantity: number }) => { if (checkDecimals(quantity)) throw errors.INVALID_ARGUMENT('quantity must have less than 13 decimal'); },
```

Comment:
We don't need this for quantity. Also on the grpc layer quantity will always be an integer, as it represents whole numbers of satoshis, so it would never fail this check.